responsibleai/responsibleai/feature_metadata.py

```diff
@@ -1,57 +1,66 @@
 # Copyright (c) Microsoft Corporation
 # Licensed under the MIT License.
 
-import warnings
 from typing import Any, Dict, List, Optional
 
 from responsibleai.exceptions import UserConfigValidationException
 
 
 class FeatureMetadata:
-    def __init__(self, identity_feature_name: Optional[str] = None,
-                 datetime_features: Optional[List[str]] = None,
+    def __init__(self,
+                 identity_feature_name: Optional[str] = None,
+                 time_column_name: Optional[str] = None,
                  categorical_features: Optional[List[str]] = None,
-                 dropped_features: Optional[List[str]] = None):
+                 dropped_features: Optional[List[str]] = None,
+                 time_series_id_column_names: Optional[List[str]] = None):
         """Placeholder class for feature metadata provided by the user.
 
         :param identity_feature_name: Name of the feature which helps to
                                       uniquely identify a row or instance
                                       in user input dataset.
         :type identity_feature_name: Optional[str]
-        :param datetime_features: List of datetime features in the
-                                  user input dataset.
-        :type datetime_features: Optional[List[str]]
+        :param time_column_name: name of datetime feature in the user input
+                                 dataset.
+        :type time_column_name: Optional[str]
         :param categorical_features: List of categorical features in the
                                      user input dataset.
         :type categorical_features: Optional[List[str]]
         :param dropped_features: List of features that were dropped by the
                                  the user during training of their model.
         :type dropped_features: Optional[List[str]]
+        :param time_series_id_column_names: List of features that are used
+                                            to uniquely identify a time
+                                            series in the user input dataset.
+        :type time_series_id_column_names: Optional[List[str]]
```

Comment:
This could be time_series_id_feature_names

Reply:
the others are all "features" rather than "feature_names" (except for the singular `identity_feature_name`) - shouldn't we stick with the pattern?
`time_series_id_features`?